spyder/app/tour.py

```diff
@@ -1388,59 +1390,114 @@ def tour_has_focus(self):
         image_path = get_image_path(icon_filename)
         image = QPixmap(image_path)
         image_label = QLabel()
-        screen = QApplication.primaryScreen()
-        device_image_ratio = screen.devicePixelRatio()
-        if device_image_ratio > 1:
-            image.setDevicePixelRatio(device_image_ratio)
-        else:
-            image_height = image.height() * 0.5
-            image_width = image.width() * 0.5
-            image = image.scaled(image_width, image_height, Qt.KeepAspectRatio,
-                                 Qt.SmoothTransformation)
+        image_height = image.height() * self.ICON_SCALE_FACTOR
+        image_width = image.width() * self.ICON_SCALE_FACTOR
+        image = image.scaled(image_width, image_height, Qt.KeepAspectRatio,
+                             Qt.SmoothTransformation)
         image_label.setPixmap(image)
 
         images_layout.addStretch()
         images_layout.addWidget(image_label)
         images_layout.addStretch()
+        if MAC:
+            images_layout.setContentsMargins(0, -5, 20, 0)
+        else:
+            images_layout.setContentsMargins(0, -8, 35, 0)
 
         # Label
+        tour_label_title = QLabel(_("Welcome to Spyder!"))
+        tour_label_title.setStyleSheet(f"font-size: {self.TITLE_FONT_SIZE}")
+        tour_label_title.setWordWrap(True)
         tour_label = QLabel(
-            _("Welcome to Spyder!<br><br>Check out our interactive tour to "
+            _("Check out our interactive tour to "
               "explore some of Spyder's panes and features."))
+        tour_label.setStyleSheet(f"font-size: {self.CONTENT_FONT_SIZE}")
         tour_label.setWordWrap(True)
+        tour_label.setFixedWidth(340)
 
         # Buttons
         buttons_layout = QHBoxLayout()
         self.launch_tour_button = QPushButton(_('Start tour'))
+        self.launch_tour_button.setStyleSheet(
+          "background-color: #3775A9;"
+          f"font-size: {self.BUTTONS_FONT_SIZE};"
+          f"padding: {self.BUTTONS_PADDING}"
+        )
         self.launch_tour_button.setAutoDefault(False)
         self.dismiss_button = QPushButton(_('Dismiss'))
+        self.dismiss_button.setStyleSheet(
+          "background-color: #60798B;"
+          f"font-size: {self.BUTTONS_FONT_SIZE};"
+          f"padding: {self.BUTTONS_PADDING}"
+        )
         self.dismiss_button.setAutoDefault(False)
+
         buttons_layout.addStretch()
         buttons_layout.addWidget(self.launch_tour_button)
+        if not MAC:
+            buttons_layout.addSpacing(10)
         buttons_layout.addWidget(self.dismiss_button)
 
         layout = QHBoxLayout()
         layout.addLayout(images_layout)
-        layout.addWidget(tour_label)
-        general_layout = QVBoxLayout()
-        general_layout.addLayout(layout)
-        general_layout.addLayout(buttons_layout)
+
+        label_layout = QVBoxLayout()
+        label_layout.addWidget(tour_label_title)
+        if not MAC:

```

Comment:
This block would be a easier to understand if it were `if MAC` and then `else`, instead of the other way around; since the brain usually works better with positives than negatives and its really unintiutive that `else` corresponds to `MAC`.